### PR TITLE
wm: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/android/wm.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# wm
+
+> Mostrar información sobre la pantalla de un dispositivo Android.
```
Comment (navarroaxel) [inline pages.es/android/wm.md:3]:
```suggestion
> Muestra información sobre la pantalla de un dispositivo Android.
```

File: pages.es/android/wm.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+> Este comando solo se puede usar a través de `adb shell`.
+> Más información: <https://adbinstaller.com/commands/adb-shell-wm-5b672b17e7958178a2955538>.
+
+- Mostrar el tamaño físico de la pantalla de un dispositivo Android:
+
+`wm {{size}}`
+
+- Mostrar la densidad física de la pantalla de un dispositivo Android:
```
Comment (navarroaxel) [inline pages.es/android/wm.md:11]:
```suggestion
- Muestra la densidad física de la pantalla de un dispositivo Android:
```

File: pages.es/android/wm.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# wm
+
+> Mostrar información sobre la pantalla de un dispositivo Android.
+> Este comando solo se puede usar a través de `adb shell`.
+> Más información: <https://adbinstaller.com/commands/adb-shell-wm-5b672b17e7958178a2955538>.
+
+- Mostrar el tamaño físico de la pantalla de un dispositivo Android:
+
+`wm {{size}}`
```
Comment (navarroaxel) [inline pages.es/android/wm.md:9]:
```suggestion
`wm {{tamaño}}`
```

File: pages.es/android/wm.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+- Mostrar la densidad física de la pantalla de un dispositivo Android:
+
+`wm {{density}}`
```
Comment (navarroaxel) [inline pages.es/android/wm.md:13]:
```suggestion
`wm {{densidad}}`
```

File: pages.es/android/wm.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# wm
+
+> Mostrar información sobre la pantalla de un dispositivo Android.
+> Este comando solo se puede usar a través de `adb shell`.
+> Más información: <https://adbinstaller.com/commands/adb-shell-wm-5b672b17e7958178a2955538>.
+
+- Mostrar el tamaño físico de la pantalla de un dispositivo Android:
```
Comment (navarroaxel) [inline pages.es/android/wm.md:7]:
```suggestion
- Muestra el tamaño físico de la pantalla de un dispositivo Android:
```

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory defects found, approved.
- This page should be reviewed by `wm` users.
- Thanks for a contribution! ❤️

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**